### DISCUSSION TITLE
How do I install homebrew on Windows

### DISCUSSION THREAD

**Discussion by Desperate4HelpIntern** [Homebrew/discussions#1950] [Getting started]
The main page claims there are instructions to install on Linux or Windows: https://docs.brew.sh/Installation but when I follow the link, its title is "Homebrew on Linux": https://docs.brew.sh/Homebrew-on-Linux.

**Comment by KunalBagaria** [comment]
It says Windows Subsystem for Linux which is not Windows, it's a compatibility layer for running Linux executables.
